README.md

```diff
@@ -21,8 +23,14 @@ and changing the command option like below:
 
 ![Configuration Screenshot](http://f.cl.ly/items/2k1C0E0e1l2Z3m1l3e1R/Settings%20-%20-Users-fcoury-Projects-crm_bliss.jpg)
 
-Or by opening your configuration file (clicking __Atom__ > __Open Your Config__)
+Or by opening your configuration file (clicking __Atom__ > __Config…__)
 and adding or changing the following snippet:
 
     'rspec':
-      'command': 'bundle exec rspec'
+      command: 'bundle exec rspec'
+
+> You may need to make a change like this if you see an error like:
+>
+> ```bash
+> bash: line 1: undefined: command not found
+> ```
```

Comment:
https://github.com/fcoury/atom-rspec/issues/82 gets a decent amount of attention; thought it would be nice to include in the README